src/main/java/com/smashingmods/alchemistry/common/block/dissolver/DissolverBlockEntity.java

```diff
@@ -54,7 +54,7 @@ public void onLoad() {
 
     @Override
     public void tick() {
-        if (!isProcessingPaused() && !getInputHandler().getStackInSlot(0).isEmpty()) {
+        if (!isProcessingPaused() && (!getInputHandler().getStackInSlot(0).isEmpty() || !internalBuffer.isEmpty())) {
```

Comment:
As spoken about, in the case of the internal buffer not being empty, I've added it into the condition so that we make sure the buffer gets processed and is never left hanging.